components/ocf-host-application/ApplicationForm.js

```diff
@@ -353,7 +365,20 @@ const ApplicationForm = ({
                           </Box>
                           <Box width={['256px', '234px', '324px']} my={2}>
                             <StyledInputFormikField
-                              label={i18nOCFApplicationFormLabel(intl, 'slug')}
+                              label={
+                                <FormattedMessage
+                                  defaultMessage="Set your URL {questionMark}"
```

Comment:
Let's keep the `{questionMark}` part out of the i18n string, it makes things more complicated for translators while the position of the question mark is never gonna change, regardless of the locale.

Reply:
Ah, yes should have caught this. I've made the changes and added a `helpText` prop to the `StyledInputField`; 

![Screenshot from 2021-11-22 08-31-35](https://user-images.githubusercontent.com/12435965/142899247-3f09cb4d-6db5-4fe2-b515-b1a8e4b2ed15.png)